src/player/index.ts

```diff
@@ -232,11 +302,36 @@ function getNextTour(): CodeTour | undefined {
   }
 }
 
-function getHeaderText(step: CodeTourStep): string {
+async function getFileContext(
+  line: number,
+  file: string,
+  uri: Uri
+): Promise<string> {
+  let label = `This step is on line ${line} in file ${file}`;
+  const document = await workspace.openTextDocument(uri);
+  if (document.languageId === "typescript") {
```

Comment:
nit: if this were a work PR I would say to use a short path/long path pattern of

if (!typescript) {return early}

<do all the complicated typescript stuff here>


rather than having so many nested blocks. This is mostly just a style thing though